nominal-streaming/src/stream.rs

```diff
@@ -669,28 +694,7 @@ impl SeriesBuffer {
             UNIX_EPOCH.elapsed().unwrap().as_nanos() as u64,
             Ordering::Release,
         );
```

Comment:
Unlike `count`, `flush_time` is not considered in this PR.

The mechanics of how these two values are accessed are a little bit different, and I haven't considered `flush_time` deeply.